hedera-node/hedera-schedule-service-impl/src/test/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleHandlerTestBase.java

```diff
@@ -0,0 +1,367 @@
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.WARN)
+class ScheduleHandlerTestBase {
+    // spotless mangles this section randomly, due to incorrect wrapping rules
+    // spotless:off
+    private static final ScheduleID ALL_SCHEDULES_ID =
+            ScheduleID.newBuilder().shardNum(12).realmNum(6).scheduleNum(1865).build();
+    // A few random values for fake ed25519 test keys
+    protected static final Bytes PAYER_KEY_HEX =
+            Bytes.fromHex("badcadfaddad2bedfedbeef959feedbeadcafecadecedebeed4acedecada5ada");
+    protected static final Bytes SCHEDULER_KEY_HEX =
+            Bytes.fromHex("feedbeadcafe8675309bafedfacecaeddeedcedebede4adaacecab2badcadfad");
+    // This one is a perfect 10.
+    protected static final Bytes ADMIN_KEY_HEX =
+            Bytes.fromHex("0000000000191561942608236107294793378084303638130997321548169216");
+    protected static final Bytes OPTION_KEY_HEX =
+            Bytes.fromHex("9834701927540926570495640961948794713207439248567184729049081327");
+    protected static final Bytes OTHER_KEY_HEX =
+            Bytes.fromHex("983470192754092657adbdbeef61948794713207439248567184729049081327");
+    protected final ScheduleID testScheduleID = ScheduleID.newBuilder().scheduleNum(100L).build();
+    protected final AccountID adminAccount = AccountID.newBuilder().accountNum(626068L).build();
+    protected final Key adminKey = Key.newBuilder().ed25519(ADMIN_KEY_HEX).build();
+    protected final AccountID scheduler = AccountID.newBuilder().accountNum(1001L).build();
+    protected final Key schedulerKey = Key.newBuilder().ed25519(SCHEDULER_KEY_HEX).build();
+    protected final AccountID payer = AccountID.newBuilder().accountNum(2001L).build();
+    protected final Key payerKey = Key.newBuilder().ed25519(PAYER_KEY_HEX).build();
+    protected final Key optionKey = Key.newBuilder().ed25519(OPTION_KEY_HEX).build();
+    protected final Key otherKey = Key.newBuilder().ed25519(OTHER_KEY_HEX).build();
+    protected final Timestamp testValidStart =
+            Timestamp.newBuilder().seconds(2281580449L).nanos(0).build();
+    protected final Instant testConsensusTime = Instant.ofEpochSecond(1656087862L, 1221973L);
+    // spotless:on
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    protected Account schedulerAccount;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    protected Account payerAccount;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    protected ReadableAccountStore accountStore;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    protected ReadableStates states;
+
+    @Mock(strictness = Mock.Strictness.LENIENT)
+    protected ReadableKVStateBase<ScheduleID, Schedule> schedulesById;
```

Comment:
Any special reason not to use `MapReadableStates` etc. here instead of mocking?

Reply:
I wasn't aware of MapReadableStates when I started on this code 3 months back (it's been rewritten several times just like the handler code as the context APIs changed).  I'll look at it.